facts/Vagrantfile

```diff
@@ -7,6 +7,7 @@ VAGRANTFILE_API_VERSION = "2"
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   config.vm.hostname = "foo.example.com"
+  config.vm.synced_folder ".", "/vagrant", type: "virtualbox"
```

Comment:
What is the goal of syncing the whole dir?

Reply:
I pushed a new `Vagrantfile` in a separate commit since i don't know how to preserve author in this situation.